NGitLab/Models/Note.cs

```diff
@@ -46,5 +46,13 @@ public class Note
         [DataMember(Name = "confidential")]
         [JsonPropertyName("confidential")]
         public bool Confidential;
+
+        [DataMember(Name = "noteable_iid")]
+        [JsonPropertyName("noteable_iid")]
+        public long NoteableIid { get; set; }
```

Comment:
Since [`noteable` is actually a misspelling of `notable`](https://en.wiktionary.org/wiki/noteable), maybe we could rename our properties appropriately?

Reply:
On second thought, seeing as there are already occurrences of that misspelling in NGitLab, we should probably stick with it for the sake of consistency. 😢